tests/test_imports.py

```diff
@@ -207,3 +207,18 @@ def test_import_alias_use_alias(v):
     check(v.unused_funcs, [])
     check(v.unused_imports, [])
     check(v.unused_vars, [])
+
+
+def test_ignore_dunder_init_py_files(v):
+    v.scan(
+        """\
+import bar
+from foo import *
+from zoo import zebra
+
+unused_var = 'genghis khan'
```

Comment:
Let's use a more neutral string here :-) What about "kangaroo" or your favorite animal?

Reply:
Hehe, pardon me. I was listening to "I get a little bit of Genghis Khan" - straight out of release notes from pogo. :p